mongoexport/mongoexport.go

```diff
@@ -69,9 +81,55 @@ type ExportOutput interface {
 	Flush() error
 }
 
-// ValidateSettings returns an error if any settings specified on the command line
+// New constructs a new MongoExport instance from the provided options.
+func New(opts Options) (*MongoExport, error) {
+	exporter := &MongoExport{
+		ToolOptions: *opts.ToolOptions,
```

Comment:
I know this is existing code, but I think it's worth updating now while you're tweaking the options: most of the other tools store ToolOptions as a pointer, probably to avoid copying the entire struct. For consistency and/or performance we should probably store a ToolOptions pointer as well in mongoexport.

Reply:
Done in new commits.